lib/index.js

```diff
@@ -108,26 +124,50 @@ FSTree.prototype = {
     }
   },
 
-  get _changes() {
-    return this._parent ? this._parent._changes : this.__changes;
+  get exclude() {
+    return this._parent ? this._parent._exclude : this._exclude;
```

Comment:
When we defer to the parent we want to use the getter, not the raw value.  Otherwise we won't get the right result if we're a projection of a projection (ie have a grandparent as well as a parent).

This brings to light that the tests I wrote for the parent stuff should really have a grandparent case as well.

anyway we need basically

```js
get exclude() {
  return this._parent ? this._parent.exclude : this._exclude;
}
```